eng/cibuild_bootstrapped_msbuild.ps1

```diff
@@ -65,65 +65,11 @@ try {
 
   if ($buildStage1)
   {
-    & $PSScriptRoot\Common\Build.ps1 -restore -build -ci -msbuildEngine $msbuildEngine /p:CreateBootstrap=true @properties
+    & $PSScriptRoot\Common\Build.ps1 -restore -build -test -ci -msbuildEngine $msbuildEngine /p:CreateBootstrap=true @properties
```

Comment:
@rainersigwald should I set `CreateBootstrap=false` here as in the .sh script?

Reply:
It may not be required but if it is it's totally fine :)